<ui version="4.0" >
 <class>pqParticleTracerPanel</class>
 <widget class="QWidget" name="pqParticleTracerPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="pqCollapsedGroup" native="1" name="streamTracerGroup" >
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="7" column="1" >
       <widget class="QLineEdit" name="TerminalSpeed" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>MaxTrackLength</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Time Step</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Integrator Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="MaxTrackLength" />
      </item>
      <item row="1" column="2" >
       <widget class="QLineEdit" name="MaximumPropagation" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1" >
       <widget class="QLineEdit" name="MaximumError" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Maximum Step Length</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>TimeStepResolution</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="InitialIntegrationStepUnit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="2" >
       <widget class="QLineEdit" name="MinimumIntegrationStep" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Vectors</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="MaximumPropagationUnit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="SelectInputVectors" />
      </item>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="TimeStepResolution" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Max. Propagation</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2" >
       <widget class="QLineEdit" name="MaximumIntegrationStep" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Minimum Step Length</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Term. Speed</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Maximum Error</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="TimeStep" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Initial Step Length</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" >
       <widget class="QComboBox" name="IntegratorType" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1" >
       <widget class="QComboBox" name="MinimumIntegrationStepUnit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="InitialIntegrationStep" />
      </item>
      <item row="10" column="1" >
       <widget class="QComboBox" name="MaximumIntegrationStepUnit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Force Reinjection Steps</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLineEdit" name="ForceReinjectionEveryNSteps" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="pqCollapsedGroup" native="1" name="seedsGroup" >
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QComboBox" name="seedType" >
        <item>
         <property name="text" >
          <string>Point Source</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Line Source</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Seed Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QStackedWidget" name="stackedWidget" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="pointSource" />
        <widget class="QWidget" name="lineSource" />
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqCollapsedGroup</class>
   <extends>QWidget</extends>
   <header>pqCollapsedGroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>seedType</sender>
   <signal>activated(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>367</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
